src/icons/general/__tests__/IconGlobeTinycon-test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { white } from '../../../styles/variables';
+
+import IconGlobeTinycon from '../IconGlobeTinycon';
```

Comment:
We don't have any naming scheme for the different usages of icons. @priyajeet is there any scheme that might make sense from your perspective? I was thinking a usage prefix like `Tinycon` which also implies associated default sizing, etc.